README.md

```diff
@@ -57,14 +57,14 @@ Mount `/tmp` to access the socket:
 
 ```json
 ...
-    "providerVaultBaseURL": "",
+    "providerVaultName": "",
     "providerKeyName": "",
     "providerKeyVersion": ""
 
 ```
-* `providerVaultBaseURL`: should look something like this: `https://yourkeyvaultname.vault.azure.net/`
-* `providerKeyName`: name of the key created in azure key vault
-* `providerKeyVersion`: key version of the key created in azure key vault
+* `providerVaultName`: name of the key vault you have created in the same resource group as your k8s cluster. If the provided key vault is not found, service will return an error.
+* `providerKeyName`: name of the key created in azure key vault. If the provided key is not found, service will create a key `providerKeyName` in the `providerVaultName` key vault.
```

Comment:
I think this shouldn't be required, if no key, then we create a key using `well known name`  on behalf of the user. this enables bring-up experience to just focus on creating keyvaults and assign the right permissions.

Reply:
my thought was to allow acs-engine to provide the `default` values for these two parameters in `azure.json`. So the default bring-up experience is the same, but if users have their own keyvault or key, then they can update these two parameters thru acs-engine as well. wdyt?

Reply:
I agree.